python/lsst/ts/linearStage/ls.py

```diff
@@ -1,41 +1,61 @@
+""" This module is for the Zaber linear stage component according to SAL specifications
+
+    This module is designed to integrate with salpytools. As such it should be installed on any machine that this module is used.
+"""
+
 # TODO: Add in Sphinx documentation
 # TODO: Finish up docstrings
 # TODO: Add module docstring
 # TODO: Add in Unit Tests
 # TODO: Add in XML
-# TODO: Add in XML for commands
 # TODO: Add in XML for events
-# TODO: Add in XML for telemetry
 
 from zaber.serial import AsciiSerial, AsciiDevice, AsciiCommand, AsciiReply
 from zaber.serial.exceptions import TimeoutError
 import logging
 from serial import SerialException
+from salpytools import salpylib
 
 logger = logging.getLogger(__name__)
```

Comment:
This a personal preference but I would rather have a logger inside the class than use a global logger like this. You can just add a `self.log = logging.getLogger(type(self).__name__)` to `__init__()` and then use `self.log` instead of `logger`... Again, a personal preference...